windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -1,79 +0,0 @@
-
-The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
-
-[id="getting-support"]
-== Getting support
-
-You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
-
-[id="wmco-2-0-0"]
-== Release notes for Red Hat Windows Machine Config Operator 2.0.0
-
-This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 2.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:0440[RHBA-2021:0440].
-
-WMCO supports self-managed clusters built using installer-provisioned infrastructure running on the following cloud providers:
-
-* Amazon Web Services (AWS)
-* Microsoft Azure
-* VMware vSphere
-
-The following Windows Server operating systems are supported in this release of the WMCO, depending on which platform your cluster is installed on:
-
-[cols="1,4",options="header"]
-|===
-
-|WMCO platform
-|Windows Server version
-
-|AWS
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
-
-|Azure
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
-
-|vSphere
-|Windows Server Semi-Annual Channel (SAC): Windows Server 1909 with link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]
-
-|===
-
-[IMPORTANT]
-====
-Running Windows container workloads is not supported for clusters in a restricted network or disconnected environment.
-====
-
-Version 2.x of the WMCO is only compatible with {product-title} 4.7.
-
-[id="wmco-2-0-0-improvements"]
-=== New features and improvements
-
-This release adds the following new features and improvements.
-
-[id="wmco-2-0-0-vsphere-support"]
-==== Support for clusters running on VMware vSphere
-
-You can now run Windows nodes on a cluster that is installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows MachineSet object on vSphere to host Windows Server compute nodes.
-
-[id="wmco-2-0-0-monitoring"]
-==== Enhanced Windows node monitoring
-
-Windows nodes are now fully integrated with most of the monitoring capabilities provided by the web console. However, it is not possible to view workload graphs for pods running on Windows nodes in this release.
-
-[id="wmco-2-0-0-known-issues"]
-=== Known issues
-
-* The filesystem graphs available in the web console do not display for Windows nodes. This is caused by changes in the filesystem queries. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930347[*BZ#1930347*])
```

Comment:
This is a known issue for WMCO 3.0.0 also and should be called out in the 3.0.0 release notes.